### ISSUE TITLE
3dep Searches IfSAR Missions Too

### ISSUE THREAD

**Issue by Jack-Hayes** [uw-cryo/coincident#24]
Coincident version: 0.1.dev40+gfa66db4.d20241108
Custom Python 3.12.5 environment

Filter out IfSAR returns from` coincident.search.search(dataset="3dep")`. Or add a boolean argument to include IFSAR results or not.

https://www.usgs.gov/centers/eros/science/usgs-eros-archive-digital-elevation-interferometric-synthetic-aperture-radar

Below is how I found the IFSAR flights:
```python
from shapely.geometry import box
global_bounds = box(-180, -90, 180, 90)
global_gdf = gpd.GeoDataFrame({"geometry": [global_bounds]}, crs="EPSG:4326")
gf_3dep_boxes = coincident.search.search(
    dataset="3dep",
    intersects=global_gdf,
    datetime=["2019", "2024"],
)
gf_ak = gpd.read_file('https://raw.githubusercontent.com/unitedstates/districts/refs/heads/gh-pages/states/AK/shape.geojson')
gf_ak_boxes = gf_3dep_boxes[gf_3dep_boxes.intersects(gf_ak.unary_union)]
# the below will contain IFSAR flights to explore
gf_ak_sites = gf_3dep_ak[gf_3dep_ak.workunit_id.isin([192357, 190677])].reset_index(drop=True)
```

**Comment by dshean** [comment]
Good call. Yes, I think we want to treat IfSAR as separate dataset than the 3DEP airborne lidar.  Also need to check acquisition time metadata and flightlines.  The IfSAR data should be distributed in tiles, which could include multiple flights per tile.  

There is value in considering the IfSAR in addition to the small subset of available 3DEP lidar data in AK, as it is a complementary airborne X-band InSAR source with ~5 m posting.  I have some existing notes from analysis of IfSAR with WV stereo DEMs that I can share.

**Comment by scottyhq** [comment]
WESM provides the 'p_method' column that labels Ifsar and the various LiDAR types:
gf.p_method.value_counts()
```
p_method
linear-mode lidar        2426
Ifsar                     403
Geiger-mode LIDAR          60
Topobathymetric LIDAR      27
Single Photon LIDAR         4
Bathymetric LIDAR           2
Flash LIDAR                 1
```

So for now, I suggest just filtering the search results:

```python
gf = gf[gf.p_method != 'Ifsar']
```

**Comment by dshean** [comment]
Oooh, that's exciting.  We should consider the lidar technology type in site selection, and look more closely at QL levels.
Curious about performance of avilable Gm options.  I assumed everything in 3DEP archive was linear-mode.
